public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
- [ ] It would be better if we delete all the files that are not related to the project. So that the codebase looks cleaner